mountpoint-s3/src/fstab.rs

```diff
@@ -24,7 +24,7 @@ impl TryFrom<FsTabCliArgs> for CliArgs {
         let cli_arg_list = fstab_cli_args.into_cli_arg_list()?;
 
         let mut cli_args = CliArgs::try_parse_from(cli_arg_list)?;
-        cli_args.foreground = true;
+        cli_args.foreground = false;
```

Comment:
Does fstab have a health check mechanism? Or is it more like fire and forget? This might cause problems if it tries to monitor spawned process as now it would deamonise itself and exit

Reply:
If we have `foreground` true, it times out waiting for the mount to complete, even though Mountpoint itself is healthy. As far as I understand, it's basically fire and forget and there's no additional health check after we've forked.